components/list/test/list-item-drag-handle.test.js

```diff
@@ -0,0 +1,94 @@
+import { DOWN, END, ENTER, ESC, HOME, LEFT, RIGHT, SPACE, TAB, UP } from '../../../helpers/keyCodes.js';
+import { expect, fixture, html, oneEvent } from '@open-wc/testing';
+import { dragActions } from '../list-item-drag-handle.js';
+
+describe('ListItemDragHandle', () => {
+
+	describe('Events to activate keyboard mode', () => {
+		let element;
+		beforeEach(async() => {
+			element = await fixture(html`<d2l-list-item-drag-handle></d2l-list-item-drag-handle>`);
+			element.focus();
+		});
+
+		it(`Expect ${dragActions.active} event results when clicked`, async() => {
```

Comment:
Nit - for BDD tests, names are usually written to read naturally starting with "it..." to describe the behavior.  The underlying expectations can change over time.

ex. "it _dispatches drag action event for [action] when clicked_"

While this is a pretty low level test, the idea behind it is that it leads us to take a more behavior driven mindset.

Reply:
That sounds awesome. Will do that